esmvalcore/cmor/_fixes/fix.py

```diff
@@ -2,15 +2,30 @@
 import importlib
 import inspect
 import os
+from functools import lru_cache
+from pathlib import Path
+
+import yaml
+
+from esmvalcore._config._config import CFG
 
 from ..table import CMOR_TABLES
 
 
+@lru_cache
+def get_variable_mappings(project, dataset):
+    DEFAULT_PATH = (Path(__file__).parents[0] / project /
+                    f"{dataset}-mappings.yml")
+    mapping_path = CFG.get(project, {}).get("mapping_path", DEFAULT_PATH)
+    with open(mapping_path, "r") as mapping_file:
```

Comment:
Must handle the cases where there is no mapping file, neither in default location, nor specified in the project definition